src/utils/icons/index.js

```diff
@@ -0,0 +1,29 @@
+// If you change the  name of this file, update iconTemplate.js
+import PropTypes from 'prop-types';
+import { css } from 'react-emotion';
+
+import { COLORS } from '../../constants';
+
+export const propTypes = {
+  inline: PropTypes.bool,
+  rotate: PropTypes.number,
+  fill: PropTypes.string,
+  className: PropTypes.string,
+  width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+};
+
+export const defaultProps = {
+  fill: COLORS.primary,
+  inline: undefined,
+  rotate: 0,
+  width: 16,
+  height: 16,
+};
+
+export const iconStyles = props => css`
+  display: ${props.inline ? "inline-block" : "block"};
+  transform: rotate(${props.rotate}deg);
+  color: ${props.fill};
+  transition: color 350ms;
```

Comment:
Do we perhaps want the transition to be something with a defaultProp which we could change too?

Reply:
Currently, there hasn't been a need to transition anything outside of the color. If we need to transition more properties, we can update the code at the time. The preference is to keep things simple until we need it, especially with how much more work is left for this project